Source/ACE.Server/WorldObjects/WorldObject.cs

```diff
@@ -246,6 +246,12 @@ private void InitializePropertyDictionaries()
 
             foreach (var x in Biota.BiotaPropertiesSpellBook)
                 BiotaPropertySpells[x.Spell] = x;
+
+            // Remove doubled up proc/use spells from spell book (incorrect data)
```

Comment:
this should not be done at runtime. the purpose of the above code is to detect those situations so that they can be fixed in the correct place in the db